Project1/Grid.h

```diff
@@ -60,16 +44,32 @@ class Grid {
 	void insertUnit(Unit& new_unit, GridCell& destination);
 	void removeUnit(Unit& unit);
 	
-	bool hasUnit(const GridCell & index) const;
-	bool hasUnit(const Unit & unit) const;
+	void moveUnit(ID& start, ID& destination);
+	void insertUnit(Unit& new_unit, ID& destination);
+
+	bool hasUnit(const GridCell& index) const;
+	bool hasUnit(const Unit & index) const;
+	bool hasUnit(const ID& index) const;
+
+	Unit* getUnit(const GridCell& index);
+	Unit* getUnit(const ID& index);
```

Comment:
should probably take IDs by value since they're just int pairs

Reply:
they are _size_t_ pairs thank you very much